### ISSUE TITLE
Bug handling gas fields in trade response

### ISSUE THREAD

**Issue by mifeng** [CoinAlpha/gateway-api#138]
After the fix in #137, `amm-arb` is closer to working again in the client. However, it appears that there is still an mismatch between the poll response and the format expected by the client. 

From the logs, the issue appears to be related to formatting of the `gasUsed` or `gasPrice` fields:
```
021-08-20 16:51:39,687 - 14921 - hummingbot.connector.connector.uniswap.uniswap_connector - INFO - Polling for order status updates of 1 orders.
2021-08-20 16:51:40,947 - 14921 - hummingbot.connector.connector.uniswap.uniswap_connector - INFO - Polling for order status updates of 1 orders.
2021-08-20 16:51:40,947 - 14921 - hummingbot.connector.connector.uniswap.uniswap_connector - ERROR - [<class 'decimal.ConversionSyntax'>]
Traceback (most recent call last):
  File "/Users/feng/Code/hummingbot/hummingbot/connector/connector/uniswap/uniswap_connector.py", line 508, in _status_polling_loop
    await safe_gather(
  File "/Users/feng/Code/hummingbot/hummingbot/core/utils/async_utils.py", line 22, in safe_gather
    return await asyncio.gather(*args, **kwargs)
  File "/Users/feng/Code/hummingbot/hummingbot/connector/connector/uniswap/uniswap_connector.py", line 394, in _update_order_status
    fee = Decimal(str(gas_used)) * Decimal(str(gas_price)) / Decimal(str(1e9))
decimal.InvalidOperation: [<class 'decimal.ConversionSyntax'>]
```
Screenshot:
<img width="1409" alt="Screen Shot 2021-08-20 at 4 52 38 PM" src="https://user-images.githubusercontent.com/1795030/130303777-d3c9c03b-af8c-424c-bbae-874d51327367.png">

Logs:
[logs_conf_amm_arb_2.log](https://github.com/CoinAlpha/gateway-api/files/7024519/logs_conf_amm_arb_2.log)

**Comment by mifeng** [comment]
Actually, I think this is caused by the same issue as #133: we are returning `gasUsed` as BigNumber when the client is expecting a decimal.

**Comment by mifeng** [comment]
Added the fix to #137
